OpenDF-UI/app/containers/HomePage/index.js

```diff
@@ -13,7 +13,6 @@ import React from 'react';
 import { FormattedMessage } from 'react-intl';
 import messages from './messages';
 import ButtonElement from '../../components/ButtonElement';
-import {Card} from '../../components/Card';
```

Comment:
In `/HomePage/index.js`, the `Card` components is used. Why remove it?

Reply:
Oh sorry, just intended to remove the imports only for the FileBrowser Page, but have mistakenly removed the imports for the HomePage as well.